### ISSUE TITLE
a11y Updates for Castos Player

### ISSUE THREAD

**Issue by edequalsawesome** [CastosHQ/Seriously-Simple-Podcasting#569]
There's a few errors/missing elements that came up when I ran the new HTML5 player through WAVE:

* Missing alt text for podcast player images
* Missing alt text for the podcast loading image
* Missing labels to identify what the `<input>` fields are in the Subscribe and Share fields
* Missing explanation text for the Copy buttons

**Comment by jonathanbossenger** [comment]
Thanks @edequalsawesome 

For my own learning, what is WAVE? Is it something that can be installed locally to test for a11y issues?

**Comment by edequalsawesome** [comment]
@jonathanbossenger It's a website, actually! [https://wave.webaim.org/](https://wave.webaim.org/)

If you put in a URL of a site (for this, I was using a test site), it will provide you with all the accessibility errors it finds, and gives explanations on what needs to be done to fix them -- it's *awesome* 😃
